test/registries.test.js

```diff
@@ -4,6 +4,16 @@
 // License text available at https://opensource.org/licenses/MIT
 
 describe('Registry', function() {
+  describe('models', function() {
```

Comment:
Please use the function name under test as the suite name.

```
describe('createModel', function() {
  // ...
});
```

We don't have a unified policy about the exact format of the test name, in some places we would have 
```js
describe('createModel()'`);
```
 or even
```js
describe('createModel(name, properties, options)');
```

Pick the style you like most :)